dts/bindings/power/state.yaml

```diff
@@ -27,4 +27,5 @@ properties:
         required: false
         description: |
             Minimum residency duration in microseconds. It is the minimum time for a
-            given idle state to be worthwhile energywise.
+            given idle state to be worthwhile energywise. It includes the time to enter
```

Comment:
can we just take the description from Linux:

"Minimum residency duration in microseconds, inclusive of preparation
          and entry, for this idle state to be considered worthwhile energy wise"